release_notes/ocp-4-13-release-notes.adoc

```diff
@@ -256,6 +256,21 @@ For more information, see xref:../networking/openshift_sdn/migrate-to-openshift-
 
 The cluster network can be expanded to support the addition of nodes to the cluster. For example, if you deployed a cluster and specified `10.128.0.0/19` as the cluster network range and a host prefix of `23`, you are limited to 16 nodes. You can expand that to 510 nodes by changing the CIDR mask on a cluster to `/14`. For more information, see xref:../networking/configuring-cluster-network-range.adoc#configuring-cluster-network-range[Configuring the cluster network range].
 
+[id="ocp-4-13-dual-stack-vmware-vsphere-clusters"]
+==== Dual-stack IPv4/IPv6 on VMware vSphere clusters
+
+On installer-provisioned vSphere clusters, you can use dual-stack networking with the IPv4 address family as the primary IP family, and the IPv6 address family as the secondary address family. For more information, see xref:../installing/installing_vsphere/installing-vsphere-installer-provisioned-network-customizations.adoc#installation-configuration-parameters-network_installing-vsphere-installer-provisioned-network-customizations[Network configuration parameters].
```

Comment:
Does this work (makes it a bit tighter):

s/IPv4 address family as the primary IP family, and the IPv6 address family as the secondary address family/IPv4 as the primary IP family, and the IPv6 as the secondary address family